test/action.yml

```
@@ -11,6 +11,9 @@ inputs:
     description: Maven settings file path
     required: true
     default: .maven/settings.xml
+  upload_reports:
+    description: Upload MUnit reports to GitHub Actions Artifacts
+    default: false
```

As we compare it with `'true'` should we 

```suggestion
    default: 'false'
```

---

I just changed it here to avoid confusion, but this default value will be a string - [ref](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault)
For the example, I won't change, as YAML treats texts as strings and we can omit the quote 🙏